CHANGELOG.md

```diff
@@ -5,7 +5,9 @@
 - Fix a crash related to behaviour changes in 1.3.0-alpha08 of the AndroidX Fragment library
 - Replace Glide with Coil in AttachmentViewHolderMedia (Fix GIFs loading issues)
 - `MessageListView.BubbleHelper`'s methods now have nullability annotations, and use primitive `boolean` values as parameters
-- Update Offline Support to the [last version](. See changes: )https://github.com/GetStream/stream-chat-android-livedata/releases/tag/0.8.6)
+- Fix method to obtain initials from user to be shown into the avatar
+- Fix method to obtain initials from channel to be shown into the avatar
```

Comment:
These were added to an existing release, instead of the _to be released_ section

Reply:
Ups, it is because previous PR #708